projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,59 @@
+package cmd
```

Comment:
do you need a help message for this command?

Reply:
yes `-n` but because that was a stretch goal i was waiting until i had fulfilled all the mandatory criteria first

Reply:
I was actually referring to the help command same as go-ls, but I was nit picking

https://github.com/bazmurphy/immersive-go-course/blob/8f332d47d0764204ebea2f89c4142b5dc381c435/projects/cli-files/go-ls/cmd/root.go#L12-L21